executors/http/src/main/java/net/opentsdb/query/execution/HttpQueryV3Result.java

```diff
@@ -90,6 +90,9 @@
 
   /** Object Mapper for serdes. */
   private final ObjectMapper mapper = new ObjectMapper();
+
+  /** flag to know if we need to process series parallely */
+  private boolean process_parallel = true;
```

Comment:
Make this volatile so the serde threads can read right values

Reply:
doesn't need to be, it'll only be read from one thread.

Reply:
Serde can now process the groupby units parallelly if needed, maybe this Query Result type won't be processed by Serde?

Reply:
It will be, but they look at this flag and take a call to process parallelly or not. This call should be from one thread always.